CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add support DOREPLIFETIME_ACTIVE_OVERRIDE for replication conditions, with the exception of TArray's this should now work the same as in native.
 - Pre and PostNetReceive are now called on an object a maximum of a single time in a given tick. This matches native much closer.
 - The GDK has been upgraded to use version 15.2.0 of SpatialOS.
+- Add helpers to the test framework - `SpawnActor`, `RequireValid`, `GetFlowPlayerController`, `RequireEqual_Enum`, and `RequireNotEqual_Enum`.
```

Comment:
Maybe would put in the internal section for now

Reply:
done